src/Http/WebUtilities/src/MultipartSection.cs

```diff
@@ -45,5 +45,6 @@ public string ContentDisposition
         /// This may not be available if the total multipart body is not seekable.
         /// </summary>
         public long? BaseStreamOffset { get; set; }
+        public long Length { get; internal set; }
```

Comment:
Why was this added?

Reply:
I had my reason but it wasn't smart, forgot to remove, Will fix this.